scripts/cluster_test_pod_template.yaml

```diff
@@ -18,7 +18,28 @@ spec:
     image: 853397791086.dkr.ecr.us-west-2.amazonaws.com/libra_cluster_test:{cluster_test_image_tag}
     imagePullPolicy: Always
     env: [{env_variables}]
-    command: [cluster-test, --deploy={image_tag} {extra_args}]
+    command:
+    - bash
+    - -c
+    - |
+      # Run kubectl proxy in the background so that cluster test can send
+      # commands to kubernetes api server through this
+      # This allows us to use http instead of https from cluster test
+      kubectl proxy &
+      for attempt in {1..10}; do
```

Comment:
Just a thought - in theory you could have this for loop in rust while cluster test starting, this could be easier / in general I would prefer to have more rust and less bash :)

Leaving this up to you @bors-libra delegate+